### PR TITLE
chore: enable Erlang/OTP 27 compatibility

### PR DESCRIPTION
**Pull request by keynslug**
See individual commits.

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (michaelklishin) [inline .github/workflows/ci.yml:10]: If we enable Erlang 27 compatibility in this PR, would it make sense to test on Erlang 27 and not stop at 26?

**Comment by michaelklishin** [comment]
@keynslug thank you for taking the time to contribute. I assume that there is a typo in https://github.com/zhongwencool/eetcd/pull/64/commits/7f6ea125b20243e66f87af5da65fdb826f54d5dd because the change does not match the commit description.

File: .github/workflows/ci.yml
Comment (keynslug) [reply .github/workflows/ci.yml]: Yes, my bad, amended the last commit. Thanks!

**Comment by michaelklishin** [comment]
I'm not sure why the workflow runs "0 checks", perhaps it's just that I don't have the permissions to see them.

All tests pass locally for me against a 3 node `etcd` cluster:

``` shell
rebar3 ct
# ===> Verifying dependencies...
# ===> Analyzing applications...
# ===> Compiling eetcd
# ===> Running Common Test suites...
# %%% eetcd_cluster_SUITE: .
# %%% eetcd_election_SUITE: .......
# %%% eetcd_kv_SUITE: .......
# %%% eetcd_lease_SUITE: .....
# %%% eetcd_lock_SUITE: ...
# %%% eetcd_maintenance_SUITE: .
# %%% eetcd_watch_SUITE: ........
# All 32 tests passed.
```

**Review by michaelklishin — APPROVED** [review]

**Comment by keynslug** [comment]
@michaelklishin, thanks for the approval! Hoping it gets merged soon.

> I'm not sure why the workflow runs "0 checks"

It says "This workflow is awaiting approval from a maintainer in #64" for me, this is probably the reason. Actions on my fork are running fine however.

https://github.com/keynslug/eetcd/actions/runs/12420631018

**Comment by michaelklishin** [comment]
> @michaelklishin, thanks for the approval! Hoping it gets merged soon.
> 
> 
> 
> > I'm not sure why the workflow runs "0 checks"
> 
> 
> 
> It says "This workflow is awaiting approval from a maintainer in #64" for me, this is probably the reason. Actions on my fork are running fine however.
> 
> 
> 
> https://github.com/keynslug/eetcd/actions/runs/12420631018

Yes, that was it. I have approved an Actions run. It is a security measure for external contributors, in particular those contributing for the first time.

**Comment by zhongwencool** [comment]
@keynslug 0.5.0 is out 🎉🎉🎉🎉🎉

**Pull request merged**
